can/__init__.py

```diff
@@ -8,7 +8,7 @@
 
 import logging
 
-__version__ = "3.3.2"
+__version__ = "3.3.3-alpha.3"
```

Comment:
Are you bumping/tagging this after merging into master?

Reply:
No I was going to do it in this branch as I've done for the pre-releases